### PR TITLE
install_grass.sh: update to GRASS GIS 8.4.0

### PR DESCRIPTION
**Pull request by neteler**
Minor script fixes and updates, in favour of GRASS GIS 8.4.0 usage.

### CODE REVIEW

File: bin/install_grass.sh [hunk 1]
```diff
@@ -50,24 +48,27 @@ fi
 USER_HOME="/home/$USER_NAME"
 
 TMP_DIR=/tmp/build_grass
-mkdir "$TMP_DIR"
+mkdir -p "$TMP_DIR"
 
+# https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable
+apt-get --quiet --yes install software-properties-common
+add-apt-repository --yes ppa:ubuntugis/ubuntugis-unstable
```
Comment (kalxas) [inline bin/install_grass.sh:55]: We are not mixing the UbuntuGIS ppa with OSGeoLive ppa, please remove.
The GRASS 8.4.0 is going to be provided in the OSGeoLive ppa

Comment (neteler) [reply bin/install_grass.sh]: I see, removed in 5a753a6040df7ac08b1654dd105a16026a19bf01

**Comment by kalxas** [comment]
Thank you, I will review and come back with comments

File: bin/install_grass.sh [hunk 2]
```diff
@@ -50,24 +48,24 @@ fi
...
+mkdir -p "$TMP_DIR"
 
 apt-get --quiet update
-apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal-grass
+apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal-grass unzip
 
-cp /usr/share/applications/grass82.desktop "$USER_HOME/Desktop/"
-chown -R $USER_NAME.$USER_NAME "$USER_HOME/Desktop/grass82.desktop"
+cp /usr/share/applications/grass8?.desktop "$USER_HOME/Desktop/Desktop GIS/"
+chown -R $USER_NAME:$USER_NAME "$USER_HOME/Desktop/Desktop GIS/grass8?.desktop"
```
Comment (kalxas) [inline bin/install_grass.sh:57]: Please revert to copy to "$USER_HOME/Desktop/"
The Desktop setup is handled in https://github.com/OSGeo/OSGeoLive/blob/master/bin/install_icons_and_menus.sh

**Review by kalxas — CHANGES_REQUESTED** [review]

File: bin/install_grass.sh (hunk 2 above)
Comment (neteler) [reply bin/install_grass.sh]: Sure, done in b55d4d69efacb039f9b6ef3f8ae36a0e579387c6

**Review by kalxas — APPROVED** [review]

**Pull request merged**